assets/stylesheets/overall.scss

```diff
@@ -33,8 +33,6 @@ body {
     position: absolute;
     bottom: 0;
     width: 100%;
-    // set the fixed height of the footer here
-    height: 30px;
```

Comment:
For me this causes the footer to overlap the **Submit comment** button unless the page is wide enough:

![image](https://github.com/os-autoinst/openQA/assets/1204189/2de5f4b0-8c97-4f0c-ab70-38734f9a0469)

On the overview the message box at the bottom is also partially obscured:

![image](https://github.com/os-autoinst/openQA/assets/1204189/65d6a40a-e0f8-4861-8300-902b44a3e2da)

Reply:
you are right

Reply:
I guess it would be ok because also on those screenshots it looks like one had to shrink the screen a lot (and also increase the zoom factor).

Maybe it would still be beneficial to make the footer a normal element (not overlapping with other elements but just taking its space on the page) in this case (e.g. using media queries or some [positioning magic](https://developer.mozilla.org/en-US/docs/Web/CSS/position) perhaps following this [guide](https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Sticky_footers)).